content/docs/2.2/scalers/openstack-metric.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Openstack Metric"
+date: 2021-03-07T17:14:29-03:00
+draft: true
+---
++++
+title = "Openstack Metric"
+layout = "scaler"
+availability = "v2.2+"
+maintainer = "Community"
+description = "Scale applications based on the metrics collected from an specific resource."
+go_file = "openstack_metric_scaler"
++++
```

Comment:
This doesn't look correct?

Reply:
Hi, thanks for noticing it! the lines 1 to 5 are actually 'dead code' I should have deleted it. I will implement all changes you pointed and improve the explanation about gnocchi integration with openstack, thanks for the review.

Reply:
Hi, I updated the file with the requested changes!